site/ko/guide/tensor.md

```diff
@@ -0,0 +1,303 @@
+# 텐서플로 텐서
+
+이름에서 알 수 있듯이, 텐서플로는 텐서를 포함한 계산을 정의하고 실행하는 프레임워크입니다.
+*텐서*는 벡터(vector)와 행렬(matrix)을 일반화한 것이고 고차원으로 확장 가능합니다.
+내부적으로 텐서플로는 텐서를 기본 데이터 타입의 n-차원 배열로 나타냅니다.
+
+텐서플로 프로그램을 작성할 때, 조작하고 전달하는 중요 객체는 `tf.Tensor` 입니다.
+`tf.Tensor` 객체는 결과적으로는 값으로 변환될 수 있는 부분적으로 정의된 계산으로 표현됩니다.
+텐서플로 프로그램은 `tf.Tensor` 객체 그래프를 만드는 것으로 먼저 시작하고,
+각각의 텐서가 다른 텐서를 기반으로 어떤 식으로 계산될 수 있는지 구체화하고,
+그 다음 그래프를 실행해서 원하는 결과를 얻게 됩니다.
+
+`tf.Tensor`는 다음과 같은 속성을 가지고 있습니다:
+
+ * 데이터 타입(data type) (예를 들어, `float32` 또는 `int32`, `string`)
+ * 쉐이프(shape)
```

Comment:
Thanks for your good suggestion.
-  '(data type)' and '(shape)' are removed.
- I agree that '변수형' is more general than '데이터 타입', so all '데이터 타입' in doc are changed.
- I would like to use '형태' rather than '텐서 형태', also all '쉐이프' in doc are changed.

Reply:
Thanks for your helpful suggestions.
First of all, the '(data type)' and '(shape)' are removed.
I agree that '변수형' is more general than '데이터 타입', so all '데이터 타입' are changed.

I would like to change 'shape' to '형태' rather than '텐서 형태', because '텐서 형태' is an appropriate translation for 'tensor shape' or 'shape of tensor'. Please let me know if you have another idea.

Reply:
In my opinion, translating 'shape' to '형태' causes reader confused. '형태' is comprehensive meaning. That's why I suggested adding '텐서'.
I think both are right translation. I think it's up to @wckim 's choice. 👍

Reply:
Thank you for your kind explanation of why you suggest it.
Depending on the context, I translated 'shape's into '형태'  and '텐서 형태'.